dispatch/bin/dispatch-admin

```diff
@@ -78,7 +87,7 @@ if __name__ == "__main__":
     try:
         func = commands[command_name]
     except KeyError:
-        print "%s isn't a valid command" % command_name
+        show_help()
```

Comment:
If we're going to show the help message when there's no arg, we should do the check before `command_name = sys.argv[1]`